action-scripts/brownie/scripts/report_gauges.py

```diff
@@ -506,7 +506,11 @@ def parse_no_reports_report(
             contractMethod = transaction.get("contractMethod", {})
             no_reports.append(
                 {
-                    "fx_name": contractMethod.get("name", "!!N/A!!"),
+                    "fx_name": (
```

Comment:
What is this change about @gosuto-inzasheru.  I think that if/else was there for a reason and it's fragile.  Can we abstract this to another PR?

Reply:
i still think it is an issue that these ci reports crash silently, it should be obvious that they didnt complete their run.

Reply:
@gosuto-inzasheru I think the issue is that the payload is wrong and the second tx is a malformed transfer.   I guess it makes sense to have it finish and post an error or something, but maybe we should be louder about the fact that something is wrong with one of the transactions.  Can we abstract this into another PR and think about how to handle it.  There are some other calldata only use_cases we should think about/see how they interact.